defold-spine/editor/src/spineext.clj

```diff
@@ -552,10 +552,10 @@
   (let [^Throwable error (if (instance? java.lang.reflect.InvocationTargetException error) (.getCause error) error)
         msg (.getMessage error)
         path (resource/resource->proj-path resource)
-        msg-missing-image "Region not found"]
+        msg-missing-image "Region not found: "]
```

Comment:
Previously, the shown error looked like this: `Atlas is missing image: : image-name`. Now we strip the duplicated `: `